File: JAVA/uberX.java
public class uberX extends car {
    public String brand;
    public String model;

    public uberX(String license, driver driver, int passenger, String brand, String model) {
        super(license, driver, passenger);
        this.brand = brand;
        this.model = model;
    }

    @Override
    public String toString() {
        return ("License: " + license + ", Driver: " + driver.name + ", Passenger: " + passenger + ", Brand: " + brand
                + ", Model: " + model);
    }
}

File: JS/uberX.js
const car = require("./car.js");

class UberX extends car {
  constructor(license, driver, passenger, brand, model) {
    super(license, driver, passenger);
    this.brand = brand;
    this.model = model;
  }

  printDataCar = () => {
    return `License: ${this.license}, Driver: ${this.driver.name}, Passenger: ${this.passenger}, Brand: ${this.brand}, Model: ${this.model}`;
  };
}

module.exports = UberX;
